qiskit/providers/models/backendproperties.py

```diff
@@ -88,12 +97,12 @@ class Gate(BaseModel):
     full description of the model, please check ``GateSchema``.
 
     Attributes:
-        qubits (list[Number]): qubits.
-        gate (str): gate.
-        parameters (Nduv): parameters.
+        qubits: qubits.
+        gate: gate.
+        parameters: parameters.
```

Comment:
Why removing the types? Are they obsolete?. Also, the explanation is not adding much :)

Reply:
We're adding type hints to the file overall. These changes are to keep type documentation consistent throughout the file. Improving the documentation for the `Gate` class is otherwise outside of the scope for this PR

Reply:
I do agree with you--- pointless docstring here

Reply:
@1ucian0 There's another PR https://github.com/Qiskit/qiskit/pull/623 to have a standardized documentation generated from the docstrings. So, there's no need to add types in the docstrings. Sphinx will extract the types from the typehints in signature and add them to the docstrings when the documentation is generated.

@lcapelluto Working on adding meaningful docstrings for this.

Reply:
This is incorrect.  We need to have a standard docstring type.  The fix in the docs is just a band-aid to make them look fine while we migrate over to a standard.  Viewing the function docstrings still how this difference.  @ajavadia needs to decide which way to go here.  Given the flow of the code, it seems that migrating over to type annotations is the way to go.